app/views/posts/_form.html.erb

```diff
@@ -46,24 +47,27 @@
 </head>
 <body>
 
+<!-- precontent in the editor -->
 <pre id="editor">
     print "Hello World"
 </pre>
 
+<!-- Run button: trigger send() function when click -->
 <div class="margin">
```

Comment:
I like your comments here, since I am actually not familiar with HTML, this is very helpful for me to understand what this code is doing.